apps/site/src/app/events/PastEvents.tsx

```diff
@@ -17,11 +18,12 @@ const PastEvents = async () => {
     <div className={styles.events}>
       {pastEvents.map((event) => (
         <EventCard
-          title={event.name}
-          startTime={event.start_time}
-          endTime={event.end_time}
-          image={event.cover.source}
-          key={event.id}
+          key={event._id}
+          title={event.title}
+          startTime={event.timeRange.start}
+          endTime={event.timeRange.end}
+          // description={event.description}
```

Comment:
Is there a reason why `description` is commented out here?

Reply:
I haven't implemented the modal yet for events, and don't want to put too much work into it yet till we work out the details of how we're going to rework the design.
So that component will eventually need the description, but just currently doesn't do anything with it - hence the comment.

Reply:
Sounds good!